### ISSUE TITLE
Data persistence

### ISSUE THREAD

**Issue by nikhilo** [Ticketmaster/techmaturity#11]
I'm trying to run the container, but not able to persist the data.
Tried invoking like so,
```
docker run -v "$PWD:/techmaturity/db/" -p 3000:3000 ticketmaster/techmaturity:1.0.0
```

**Comment by vigneshjb** [comment]
@nikhilo : Thank you for trying out Techmaturity. We currently don't support `mysql` but we do support `pg`. I just tried running a similar Docker run command to connect with a Postgres database and everything worked just fine. Can you please share some more info. 

Also, we have released Version 2.0.0. Please try that on your next attempt.

**Comment by vigneshjb** [comment]
@nikhilo : I just got a confirmation that the custom database.yml file fixed this. Please refer https://github.com/Ticketmaster/techmaturity/issues/9. If you still have issues, please feel free to re-open this issue.
